Documentation/Concepts/event_store.md

```diff
@@ -2,90 +2,189 @@
+
+The streams `StreamId` is added to the collections name, eg. a stream with the id of `323bcdb2-5bbd-4f13-a7c3-b19bc2cc2452` would be in a collection called `stream-323bcdb2-5bbd-4f13-a7c3-b19bc2cc2452`.
+
+```json
+{
+    // same as an Event in the "event-log" + Partition
+    "Partition": "UUID",
+}
+```
+
+### `public-stream`
+
+The same as a [`stream`]({{< ref "#stream" >}}), except only for [Public Stream]({{< ref "streams#public-vs-private-streams" >}}) with the `public` prefix in collection name. Public streams can only exist on the default [scope]({{< ref "#scope" >}}).
+
+### `stream-definitions`
+
+This collection contains all [Filters]({{< ref "event_handlers_and_filters#filters" >}}) registered with the Runtime.
+
+Filters defined by an [Event Handler]({{< ref "event_handlers_and_filters#event-handlers" >}}) have a type of `EventTypeId`, while other filters have a type of `Remote`.
+
+```json
+{
+    // id of the Stream the Filter creates
+    "_id": "UUID",
+    "Partitioned": "bool",
+    "Public": "bool",
+    "Filter": {
+        "Type": "string",
+        "Types": [
+            // EventTypeIds to filter into the stream
+        ]
+    }
+}
+```
+
+### `stream-processor-states`
+
+This collection keeps track of all [Stream Processors]({{< ref "streams#stream-processor" >}}) and their state. Partitioned streams will have a `FailingPartitions` property for tracking the fail information per partition.
+
+```json
+{
+    "SourceStream": "UUID",
+    "EventProcessor": "UUID",
+    "Position": "decimal",
+    "LastSuccesfullyProcessed": "date",
+    // failure tracking information
+    "RetryTime": "date",
+    "FailureReason": "string",
+    "ProcessingAttempts": "int",
+    "IsFailing": "bool
+}
+```
+
+### Scope
+
+Events that came over the [Event Horizon]({{< ref "event_horizon" >}}) need to be put into a scoped collection so they won't be mixed with the other events from the system.
+
+Scoped collections work in the same way as other collections, except:
+- You can't have [Public Streams]({{< ref "streams#public-stream" >}}) or [Aggregates]({{< ref "aggregates" >}})
+- There is a [`subscription-states`]({{< ref "#subscription-states" >}}) collection for [Subscription]({{< ref "event_horizon#subscription" >}})
+- Scoped collections have a `x-scopeID-` prefix in their names
+
+<!-- The default alert shortcode wouldn't work properly inside the tab so I copied the alert HTML here -->
+<div class="alert alert-info" role="alert">
+    <h4 class="alert-heading">Default scope</h4>
+    Technically all collections are scoped, with the default scopeID being <code>00000000-0000-0000-0000-000000000000</code>.
+    This is left out of the naming to make the event store more readable. When we talk about scoped concepts, we always refer to non-default scopes.
+</div>
```

Comment:
This should maybe be out of the code tab thingy?

Reply:
Kinda yeah, I just thought that the `x-scopeID-` part and the default scope baing an empty Guid is kinda a specific implementation detail. Or maybe the scoped part could be put up before the whole sturcure thing and then in the beginning of the strictire part I mention the scopes hmmm I'll get to it

Reply:
Yes do that 👍